features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -923,10 +923,17 @@
 <!--        <bundle>mvn:org.openhab.addons.bundles/org.openhab.extensionservice.marketplace.automation/${project.version}</bundle>-->
     </feature>
 
-    <feature name="openhab-misc-mqttbroker" description="Embedded MQTT Broker" version="${project.version}">
+    <feature name="openhab-misc-mqttbroker" description="MQTT Broker Moquette" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-transport-mqtt</feature>
-        <bundle>mvn:org.openhab.io/org.openhab.io.mqttembeddedbroker/${project.version}</bundle>
+        <bundle dependency="true">wrap:mvn:io.netty/netty-common/4.1.34.Final$Bundle-Name=Netty%20Common&amp;Bundle-SymbolicName=io.netty.netty-common&amp;Bundle-Version=4.1.34</bundle>
```

Comment:
It won't work because of the missing prerequisite. :wink:

Reply:
I thought we are adding it to `openhab-runtime-base`?

Reply:
It hasn't been merged yet so it will be broken until so. Why not just add the prerequisite and then we'll remove it again from all features once https://github.com/openhab/openhab-core/pull/676 is merged? At least that way there is a chance everything keeps working.

Reply:
done

Reply:
Good! Spaces instead of a tab would have been even nicer.

Reply:
Argh.. I have used the github editor and assumed it is smarter then that. Will fix it in a later PR.
Thanks.